rubocop-bsm.gemspec

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+  spec.name          = 'rubocop-bsm'
+  spec.version       = '0.5.0'
```

Comment:
@dim Did not pay attention before, but why initial release is 0.5? Just curious.